apps/judicial-system/backend/src/app/modules/subpoena/subpoena.controller.ts

```diff
@@ -55,14 +63,40 @@ import { Subpoena } from './models/subpoena.model'
   CaseReadGuard,
   DefendantExistsGuard,
 )
-@Controller('api/case/:caseId/defendant/:defendantId/subpoena')
-@ApiTags('subpoenas')
 export class SubpoenaController {
   constructor(
     private readonly pdfService: PdfService,
+    private readonly subpoenaService: SubpoenaService,
     @Inject(LOGGER_PROVIDER) private readonly logger: Logger,
   ) {}
 
+  @RolesRules(
+    prosecutorRule,
+    prosecutorRepresentativeRule,
+    districtCourtJudgeRule,
+    districtCourtAssistantRule,
+    districtCourtRegistrarRule,
+  )
+  @Get(':subpoenaId')
+  @UseGuards(SubpoenaExistsGuard)
+  @ApiOkResponse({
+    type: Subpoena,
+    description: 'Gets the subpoena for a given defendant',
+  })
+  getSubpoena(
```

Comment:
This endpoint takes care of retrieving subpoenas from the client. The subpoena is refreshed from the police system as needed.